rasa/cli/x.py

```diff
@@ -112,10 +112,21 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
+    custom_url = endpoints.model.url
+
+    if custom_url and custom_url != rasa_x_url:
+        logger.info(
+            f"Ignoring url '{custom_url}' from 'endpoints.yml' and using "
+            f"{rasa_x_url}/projects/default/models/tag/production instead"
+        )
+
     endpoints.model = EndpointConfig(
-        "{}/projects/default/models/tags/production".format(rasa_x_url),
+        f"{rasa_x_url}/projects/default/models/tags/production",
```

Comment:
I'd define a variable for the whole url before so you can reuse it in your `info` message and here.